### PR TITLE
Update export-search-results.md

### PR DESCRIPTION
**Pull request by Sokarpi**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/export-search-results.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ Here's more information about exporting search results.
 
   - You can export a maximum of 2 TB of data from a single Content Search. If the search results are larger than 2 TB, consider using date ranges or other types of filters to decrease the total size of the search results.
   
-  - Your organization can export a maximum of 2 TB of data during a single day.
+  - Your organization can export a maximum of 2 TB of data during a single day. The quota will reset daily at 12:00AM UTC.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/export-search-results.md:189]:
```suggestion
  - Your organization can export a maximum of 2 TB of data during a single day. The quota will reset daily at midnight UTC.
```

Comment (JohanFreelancer9) [reply microsoft-365/compliance/export-search-results.md:189]: Suggestion in accordance with Microsoft Style Guide.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by Sokarpi** [comment]
Thank you Mark!

Sonya Karpishpan
Support Escalation Engineer | Office 365 Security & Compliance | Equivio
sonya.karpishpan@microsoft.com<mailto:sonya.karpishpan@microsoft.com>| +1 301 771 8687 | Manager: Ryan Jones ryanjon@microsoft.com<mailto:ryanjon@microsoft.com>
[MSFT_logo_Gray DE sized SIG1.png]

Join our Preview Programs<http://aka.ms/mip-preview>
Resources: Documentation<https://aka.ms/MIPCDocs>, MIP&C Blog<https://aka.ms/MIPblog>, Tech Community<https://aka.ms/ComplianceTechCommunity>, Yammer<https://aka.ms/askMIPteam>
MIP product feedback via UserVoice<https://aka.ms/MIPCUserVoice> | Compliance product feedback via UserVoice<http://aka.ms/CompUV>
Check out the sample files and documents on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnihendle%2FMIP-Comp%2Ftree%2Fmaster%2FMIP%2FWebinars%2FUnified%2520Labeling%2520Migration&data=02%7C01%7CSonya.Karpishpan%40microsoft.com%7C5bd37595404b4df614f308d7bc5d1658%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637184980306867818&sdata=9GX5LmJvnFb8%2BJe9Sh%2Bn5CrqDUrl3uSQymkZg2FEe9U%3D&reserved=0>

Follow us on Twitter @MipNews

**Pull request merged**